selfservice/strategy/profile/two_step_registration.go

```diff
@@ -194,8 +205,8 @@ func (s *Strategy) displayStepTwoNodes(ctx context.Context, w http.ResponseWrite
 	regFlow.UI.Messages.Add(text.NewInfoSelfServiceChooseCredentials())
 
 	regFlow.UI.Nodes.Append(node.NewInputField(
-		"method",
-		"profile:back",
+		"action",
+		"back",
```

Comment:
This is really the only potential breaking change. If someone relies on `profile:back` being available, then this breaks. I checked in Ory Elements where there is no reference on this. In Ory Network neither. It was also not documented. So should be fine, hopefully?

Reply:
I think this should be fine. I invented it for two-step logins, so it really should only have been referenced inside Kratos.